pages/getting-started/install-memgraph.mdx

```diff
@@ -19,10 +18,7 @@ Lab](/data-visualization) and a complete set of algorithms within a
 With the [Docker](https://docs.docker.com/engine/install/) running in the
 background, run the following command in the console:
 
-<CodeSnippet code="docker run -p 7687:7687 -p 7444:7444 -p 3000:3000 --name memgraph memgraph/memgraph-platform" page="install-memgraph"/>
-
-You can stop the instance with `docker stop memgraph` and start it with `docker
-start memgraph` commands.
+<CodeSnippet code="curl https://install.memgraph.com | sh" page="getting-started/first-steps-with-memgraph"/>

```

Comment:
Again, please be careful not to change page parameter in the CodeSnippet component. It is important it stays the same. Besides that, split to Linux/MacOS & Windows will be necessary here as well.

Reply:
Done.